openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -137,6 +137,7 @@ def __post_init__(self):
     ),
     ProviderWorkflow(
         ingester_class=FreesoundDataIngester,
+        pull_timeout=timedelta(hours=48),  # TODO: Remove after full Freesound run
```

Comment:
I'm not sure 48 hours will be sufficient 🤔 The 2022-11-01 run in production (the `skip_ingestion_errors` run) timed out after 24 hours with the delay at 1 second. Can we extend this further? I think it could realistically take a week, maybe two to be safe as long as we check in on it regularly.

Reply:
Ahh, thanks for that extra context! Yea I think 240 hours (10 days) would be a good place to start then!